R/util.R

```diff
@@ -188,15 +188,24 @@ yaml_load <- function(string) {
   yaml::yaml.load(string, handlers = handlers)
 }
 
+get_orderly_file_names <- function(path) {
+  c(sprintf("%s.R", basename(path)), "orderly.R")
+}
 
-file_exists <- function(..., workdir = NULL) {
+file_exists <- function(..., workdir = NULL, single_file = FALSE) {
   files <- c(...)
   if (!is.null(workdir)) {
     assert_scalar_character(workdir)
     owd <- setwd(workdir) # nolint
     on.exit(setwd(owd)) # nolint
   }
-  fs::file_exists(files)
+  if (single_file) {
```

Comment:
I think you have two different functions here masquerading as one with an option?